tools/unitctl/README.md

```diff
@@ -68,6 +101,11 @@ unitd instance [pid: 79489, version: 1.32.0]:
 ```
 
 ### Start a new Unit process via docker
```

Comment:
```suggestion
### Start a new Unit process via Docker
```
Per the Docker guidelines, as this is a product, it should be title case: https://docs.docker.com/contribute/style/grammar/#best-practice-2